js/src/common/Translator.ts

```diff
@@ -4,34 +4,44 @@ import username from './helpers/username';
 import extract from './utils/extract';
 
 export default class Translator {
-  constructor() {
-    /**
-     * A map of translation keys to their translated values.
-     *
-     * @type {Object}
-     * @public
-     */
-    this.translations = {};
+  /**
+   * A map of translation keys to their translated values.
+   */
+  translations = {};
+
+  /**
+   * The underlying ICU MessageFormatter util.
+   *
+   * @ts-ignore
```

Comment:
What's this ignoring?

Reply:
Hmm, it's not erroring anymore. Weird...